src/test/resources/features/Practice.feature

```diff
@@ -69,34 +69,34 @@ Feature: Practice test cases
 
     Examples:
       | First Name | Last Name | email                                      |
-      | SUPPLIER   | TEST      | AUTO_company.apartment@yopmail.com         |
-      | SUPPLIER   | TEST      | AUTO_accommodation.discount@yopmail.com    |
-      | SUPPLIER   | TEST      | AUTO_accommodation.assistance@yopmail.com  |
-      | SUPPLIER   | TEST      | AUTO_rent.subsidy@yopmail.com              |
-      | SUPPLIER   | TEST      | AUTO_temporary.residence@yopmail.com       |
-      | SUPPLIER   | TEST      | AUTO_company.car@yopmail.com               |
-      | SUPPLIER   | TEST      | AUTO_car.rentaldiscount@yopmail.com        |
-      | SUPPLIER   | TEST      | AUTO_fuel.card@yopmail.com                 |
-      | SUPPLIER   | TEST      | AUTO_shuttle.service@yopmail.com           |
-      | SUPPLIER   | TEST      | AUTO_public.transitpass@yopmail.com        |
-      | SUPPLIER   | TEST      | AUTO_mercedes.benz@yopmail.com             |
-      | SUPPLIER   | TEST      | AUTO_bmw.car@yopmail.com                   |
-      | SUPPLIER   | TEST      | AUTO_audi.car@yopmail.com                  |
-      | SUPPLIER   | TEST      | AUTO_toyota.camry@yopmail.com              |
-      | SUPPLIER   | TEST      | AUTO_honda.accord@yopmail.com              |
-      | SUPPLIER   | TEST      | AUTO_iphone.15promax@yopmail.com           |
-      | SUPPLIER   | TEST      | AUTO_iphone.15pro@yopmail.com              |
-      | SUPPLIER   | TEST      | AUTO_samsung.galaxys21@yopmail.com         |
-      | SUPPLIER   | TEST      | AUTO_samsung.galaxynote20ultra@yopmail.com |
-      | SUPPLIER   | TEST      | AUTO_samsung.galaxyzfold@yopmail.com       |
-      | SUPPLIER   | TEST      | AUTO_mailbox.access@yopmail.com            |
-      | SUPPLIER   | TEST      | AUTO_high.performance@yopmail.com          |
-      | SUPPLIER   | TEST      | AUTO_graphics.optimized@yopmail.com        |
-      | SUPPLIER   | TEST      | AUTO_macbook.pro@yopmail.com               |
-      | SUPPLIER   | TEST      | AUTO_macbook.air@yopmail.com               |
-      | SUPPLIER   | TEST      | AUTO_dual.use@yopmail.com                  |
-      | SUPPLIER   | TEST      | AUTO_sodexo.card@yopmail.com               |
-      | SUPPLIER   | TEST      | AUTO_multinet.card@yopmail.com             |
-      | SUPPLIER   | TEST      | AUTO_setcard.card@yopmail.com              |
-      | SUPPLIER   | TEST      | AUTO_ticket.card@yopmail.com               |
+      | Supplier   | Test      | AUTO_company.apartment@yopmail.com         |
```

Comment:
Let's use it like that;
| Company            | Apartment | AUTO_company.apartment@yopmail.com             |
| Accommodation | Discount    | AUTO_accommodation.discount@yopmail.com    |